packages/ui/src/components/va-progress-circle/const.ts

```diff
@@ -0,0 +1,15 @@
+import { ArrayElementType } from '../../utils/types/array'
+
+export { defaultSizes as sizes, DefaultSizes as Sizes } from '../../composables'
+
+export const variables = [
+  'position',
```

Comment:
Position seems to be useless.

Reply:
It was in the variables that were defined so I added keys for all of the current variables.